### PR TITLE
Changes the icon; adds create date

### PR DESCRIPTION
**Pull request by sofyakurilova**
Screenshots after changes
8huopSTLjRWscAW
ACBfKQYL2EcFMZq

Tests
9TqiZfwhEjGNdbj (1)

### CODE REVIEW

File: modules/ui/src/app/pages/risk-assessment/profile-item/profile-item.component.html [hunk 1]
```diff
@@ -14,17 +14,30 @@
...
-  <p
-    class="profile-item-name"
-    role="button"
-    tabindex="0"
-    (click)="profileClicked.emit(profile)"
-    (keydown.enter)="profileClicked.emit(profile)">
-    {{ profile.name }}
-  </p>
+  <mat-icon
+    *ngIf="profile.status === ProfileStatus.DRAFT"
```
Comment (OlgaMardvilko) [inline modules/ui/src/app/pages/risk-assessment/profile-item/profile-item.component.html:24]: There is also can be the value as "Expired" that has different icon.
The link to the doc: https://docs.google.com/document/d/1XI-Jv-tEHRc9x9TOy9SBKixYSai6i9YmZYm0LFzkVv4/edit?resourcekey=0-LywLMRKxfE-5Ax9R0QbeaQ&tab=t.0#heading=h.94byhq6mp4bc

Should we also add it or there is another ticket for it?

File: modules/ui/src/app/pages/risk-assessment/profile-item/profile-item.component.html [hunk 2]
```diff
@@ -14,17 +14,30 @@
...
+    <p
+      class="profile-item-name"
+      role="button"
+      tabindex="0"
+      (click)="profileClicked.emit(profile)"
+      (keydown.enter)="profileClicked.emit(profile)">
+      {{ profile.name }}
+    </p>
+    <p class="profile-item-created">
+      {{ profile.created | date: 'dd MMM yyyy' }}
```
Comment (OlgaMardvilko) [inline modules/ui/src/app/pages/risk-assessment/profile-item/profile-item.component.html:38]: Based on the UI we also display this value on the download zip report. 
The link to the design: https://www.figma.com/design/C8U3CASvjyHV8Bf5S4ZMNz/Testrun-UI?node-id=4218-36310&m=dev
Should we also add it or there is another ticket?

Comment (sofyakurilova) [reply modules/ui/src/app/pages/risk-assessment/profile-item/profile-item.component.html:38]: I think here, thanks

File: modules/ui/src/app/pages/risk-assessment/profile-item/profile-item.component.html (hunk 1 above)
Comment (sofyakurilova) [reply modules/ui/src/app/pages/risk-assessment/profile-item/profile-item.component.html:24]: It'll be added later

**Review by OlgaMardvilko — APPROVED** [review]

**Pull request merged**